sampleserver/sampleserver.go

```diff
@@ -365,11 +393,20 @@ func runSCP(port string, dir string, remoteAEs map[string]string, tlsConfig *tls
 			data []byte) dimse.Status {
 			return ss.onCStore(transferSyntaxUID, sopClassUID, sopInstanceUID, data)
 		},
+		CStoreStream: func(connState netdicom.ConnectionState, transferSyntaxUID string,
```

Comment:
Pls add a flag to sample server to turn this on/off. So that we can check that it still works with `CStore`

Reply:
it's added